zotero2remarkable_bridge/rmapi_shim.py

```diff
@@ -14,15 +14,16 @@
     raise FileNotFoundError("Could not find 'rmapi' on your system PATH.")
```

Comment:
```
from functools import cache

...

@cache
def get_rmapi_location() -> str:
    """Get rmapi location or raise error"""
    location = shutil.which("rmapi") or shutil.which('./rmapi')
    if location is None:
        raise FileNotFoundError("Could not find 'rmapi' on your system PATH.")
    return location
```

If you then replace all rmapi_location with `get_rmapi_location`, then mypy will report no more errors for `rmapi_shim.py` :)